### PR TITLE
Create gizmo.py

### PR DESCRIPTION
**Pull request by arturosme**

### CODE REVIEW

File: gizmo.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    
+    return
+
+
+def relative_path(subject_identifiers):
+    
+    rel_path = './subjects/mock_recording_'
+    ext = '.rec'
+    
+    file_list = [rel_path + elem + ext for elem in subject_identifiers]
```
Comment (wmvanvliet) [inline gizmo.py:27]: take a look at [f-strings](https://realpython.com/python-f-strings/):

```python
file_list = [f'./subjects/mock_recording_{elem}.rec' for elem in subject_identifiers]
```

File: gizmo.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      
+        self.name = name
+      
+        return
+    
+    def speak(self):
+        
+        print(self.name)
+        
+        return
```
Comment (wmvanvliet) [inline gizmo.py:44]: why do you end each function with `return`?

File: gizmo.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+          Matrix with dimension 2 containing multiplication table
+          up to 12 with values set to zero according to zero_out_multiples
+      """
+      mat = np.outer(np.arange(1,13,1),np.arange(1,13,1))
+      
+      if zero_out_multiples is None:
+          return mat
+      else:
+          mat[mat % zero_out_multiples == 0] = 0
+          return mat
```
Comment (wmvanvliet) [inline gizmo.py:73]: fantastic implementation!

File: gizmo.py [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        yield n1
+        it += 1
+        n1, n2 = n2, n1 + n2
+
+def get_fibonacci_sequence(n):
+    
+    lst_fib = []
+    fib = generate_fibonacci_sequence(n)
+    for ii in range(0, n):
+        lst_fib.append(next(fib))
```
Comment (wmvanvliet) [inline gizmo.py:92]: no need for this loop. You can just write `lst_fib = list(fib)`

File: gizmo.py (hunk 1 above)
Comment (arturosme) [reply gizmo.py:27]: Thanks for the advice! I was not familiar with this type of string.

File: gizmo.py (hunk 2 above)
Comment (arturosme) [reply gizmo.py:44]: I am used to close/finish functions with something (from Fortran and C I suppose) and I was under the impression that, as a safeguard, it was good practice to finish the function with `return`.

Comment (wmvanvliet) [reply gizmo.py:44]: it's unusual in Python, but I kind of like it.

**Comment by wmvanvliet** [comment]
Great job on this. See you at the course!

**Pull request closed without merging**